spring-cloud-skipper-platform-cloudfoundry/src/main/java/org/springframework/cloud/skipper/deployer/cloudfoundry/CloudFoundryReleaseAnalyzer.java

```diff
@@ -71,12 +73,14 @@ public ReleaseAnalysisReport analyze(Release existingRelease, Release replacingR
 					emptyPropertiesDiff, emptyPropertiesDiff, emptyPropertiesDiff, propertiesDiff, emptyPropertiesDiff);
 			applicationManifestDifferences.add(applicationManifestDifference);
 		}
-		return createReleaseAnalysisReport(existingRelease, replacingRelease, applicationManifestDifferences);
+		return createReleaseAnalysisReport(existingRelease, replacingRelease, applicationManifestDifferences,
+				Arrays.asList(existingApplicationManifest.getName()), isForceUpdate);
```

Comment:
I am not sure this works, isn't `Arrays.asList(existingApplicationManifest.getName())` going to return just a single name instead of 'all application names'?

Reply:
In this case, we support only the single application for CF based release manager. Hence, we return just the app name in use.